src/main/java/taxi/service/DriverServiceImpl.java

```diff
@@ -38,4 +38,10 @@ public Driver update(Driver driver) {
     public boolean delete(Long id) {
         return driverDao.delete(id);
     }
+
+    @Override
+    public Driver findByLogin(String login) {
```

Comment:
the same: public Optional<Driver> findByLogin(String login)

Reply:
the same situation)